entry.go

```diff
@@ -577,12 +606,17 @@ func StartReverse(config *EntryConfiguration, wallet *nkn.Wallet) error {
 				}
 				continue
 			}
-
-			data := make([]byte, n)
-			copy(data, buffer)
+			if bytes.Equal(clientBuffer[:udpconn.PrefixLen], []byte{0, 0, 0, 0}) {
```

Comment:
What if a data packet starts with this prefix?

Reply:
`prefix` begin with connID and it couldn't be 0

> What if a data packet starts with this prefix?